### ISSUE TITLE
Search result pages with video links are raising "NoMethodError: undefined method `inner_text' for nil:NilClass" 

### ISSUE THREAD

**Issue by danneu** [postmodern/gscraper#2]
I can replicate this in irb by trying to resolve a page of a query with video results like "michael jackson":

```
irb(main):001:0> require 'gscraper'
=> true
irb(main):002:0> q = GScraper::Search.query(:query => "michael jackson")
irb(main):003:0> q.page(1)
=> NoMethodError: undefined method `inner_text' for nil:NilClass
```

It might be a good idea to treat li.g elements that also contain "videobox" in the class differently or just ignore them by default since it makes more sense not to make them increment the rank counter.

**Comment by postmodern** [comment]
It is probably time for me to create different Result Classes for the new types of content.

**Comment by danneu** [comment]
Been looking into it further and it's not just video content.

**Comment by postmodern** [comment]
Noticing some specs are failing too. Will have to carve out some time next weekend to dive in.

**Comment by Blue-Dog-Archolite** [comment]
Any further work on this issue?

**Comment by postmodern** [comment]
Alright, I think I have fixed this in 8c8fd543b658290a6406a61d715ef8397dc4ab36, and will be released in 0.4.0 (probably today).
